From a4f7916ee10179363758f8fa98cb0f0cf4fe12e6 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 14 Mar 2013 14:31:08 +0800
Subject: [PATCH 175/281] ENGR00253848 Fix UI issue in Launcher

No highlight line issue under JB4.2.2's status bar, so to revert below commit
Revert "ENGR00241208 Enlarge the navigation bar height"
This reverts commit 39ff45bfc1959049ad968c6ccf1624ec66b865ce.

Overlay Launcher2's workspace_width_gap_land setting to 32dp, which fix
app icon and widget placement cross the boundary of Launcher2 screen

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 .../frameworks/base/core/res/res/values/dimens.xml | 24 ----------------------
 .../frameworks/base/core/res/res/values/dimens.xml | 24 ----------------------
 .../apps/Launcher2/res/values-sw720dp/dimens.xml   | 20 ++++++++++++++++++
 3 files changed, 20 insertions(+), 48 deletions(-)
 delete mode 100644 sabreauto_6q/overlay/frameworks/base/core/res/res/values/dimens.xml
 delete mode 100644 sabresd_6dq/overlay/frameworks/base/core/res/res/values/dimens.xml
 create mode 100644 sabresd_6dq/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml

diff --git a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/dimens.xml b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/dimens.xml
deleted file mode 100644
index 7532ad1..0000000
--- a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/dimens.xml
+++ /dev/null
@@ -1,24 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-/* //device/apps/common/assets/res/any/dimens.xml
-**
-** Copyright 2006, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
-<resources>
-    <dimen name="navigation_bar_height">55dp</dimen>
-    <!-- Height of the bottom navigation bar in portrait; often the same as @dimen/navigation_bar_height -->
-    <dimen name="navigation_bar_height_landscape">55dp</dimen>
-</resources>
diff --git a/sabresd_6dq/overlay/frameworks/base/core/res/res/values/dimens.xml b/sabresd_6dq/overlay/frameworks/base/core/res/res/values/dimens.xml
deleted file mode 100644
index 7532ad1..0000000
--- a/sabresd_6dq/overlay/frameworks/base/core/res/res/values/dimens.xml
+++ /dev/null
@@ -1,24 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-/* //device/apps/common/assets/res/any/dimens.xml
-**
-** Copyright 2006, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
-<resources>
-    <dimen name="navigation_bar_height">55dp</dimen>
-    <!-- Height of the bottom navigation bar in portrait; often the same as @dimen/navigation_bar_height -->
-    <dimen name="navigation_bar_height_landscape">55dp</dimen>
-</resources>
diff --git a/sabresd_6dq/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml b/sabresd_6dq/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml
new file mode 100644
index 0000000..d55a391
--- /dev/null
+++ b/sabresd_6dq/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2011 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+
+<resources>
+    <dimen name="workspace_width_gap_land">32dp</dimen>
+</resources>
+
-- 
1.8.0

